shared/js/background/classes/site.es6.js

```diff
@@ -103,24 +104,56 @@ class Site {
      * determine if domain is a special page
      *
      * returns: a useable special page description string.
-     *          or false if not a special page.
+     *          or null if not a special page.
      */
-    specialDomain () {
-        if (this.domain === 'extensions') { return 'extensions' }
+    getSpecialDomain () {
+        const extensionId = browserWrapper.getExtensionId()
+        const url = this.url
+        let domain = this.domain
 
-        if (window.chrome && this.domain === chrome.runtime.id) { return 'options' }
+        if (url === '') {
+            return 'new tab'
+        }
```

Comment:
In the previous version, we only did that if the browser is FF. I can't imagine a case in which the URL will be empty but we wouldn't want to consider it a new tab.